ui/app/src/components/CommitResolutionDialog/CommitResolutionDialog.tsx

```diff
@@ -19,20 +19,19 @@ import Dialog from '@material-ui/core/Dialog';
 import CommitResolutionDialogContainer, {
   CommitResolutionDialogContainerProps
 } from './CommitResolutionDialogContainer';
+import { useOnClose } from '../../utils/hooks/useOnClose';
 
 export default function CommitResolutionDialog(props: CommitResolutionDialogContainerProps) {
   const { open, onClose } = props;
   const [disableQuickDismiss, setDisableQuickDismiss] = useState(false);
+  const onCloseHandler = useOnClose();
+
+  const handleClose = (event, reason) => {
+    onCloseHandler(onClose, reason, disableQuickDismiss, disableQuickDismiss);
+  };
```

Comment:
The idea was that the hook returns the handler directly; why not send these arguments directly to the hook?